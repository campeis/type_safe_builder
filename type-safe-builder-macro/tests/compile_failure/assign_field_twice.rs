use type_safe_builder_macro::Builder;

#[derive(Builder)]
#[builder(multi)]
struct StructToBuild {
    #[builder(single)]
    field: i64,
}

fn main() {
    let _ = StructToBuildBuilder::builder().field(1).field(2).build();
}
