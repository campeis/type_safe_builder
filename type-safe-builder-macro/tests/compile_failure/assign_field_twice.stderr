error[E0599]: no method named `field` found for struct `StructToBuildBuilderState<true>` in the current scope
 --> tests/compile_failure/assign_field_twice.rs:9:54
  |
3 | #[derive(Builder)]
  |          ------- method `field` not found for this struct
...
9 |     let _ = StructToBuildBuilder::builder().field(1).field(2).build();
  |                                                      ^^^^^--- help: remove the arguments
  |                                                      |
  |                                                      field, not a method
